main.go

```diff
@@ -33,9 +33,11 @@ const (
 	customizedImagesFile = "images/customized-images.yaml"
 	defaultPlatform      = "linux/amd64"
 	dockerTransport      = "docker://"
-	quayURL              = "quay.io/giantswarm"
-	aliyunURL            = "giantswarm-registry.cn-shanghai.cr.aliyuncs.com/giantswarm"
 	filteredFileSuffix   = ".filtered"
+
+	aliyunURL = "giantswarm-registry.cn-shanghai.cr.aliyuncs.com/giantswarm"
+	azureURL  = "gsoci.azurecr.io"
+	quayURL   = "quay.io/giantswarm"
```

Comment:
for the time being, we also need to push to the old ACR, `giantswarm.azurecr.io`

Reply:
Can we also make this configurable? We will have to change it a few times in the coming months, recompiling each time doesn't seem very appealing...

Reply:
Sure, I can make it more configurable.

Reply:
Actually, it would require similar effort. Let's talk offline